cli/assets/templates/assemblyscript/src/main.ts

```diff
@@ -12,12 +12,12 @@ const smiley = memory.data<u8>([
 ]);
 
 export function update (): void {
-    store<u16>(w4.DRAW_COLORS, 2);
+    store<u16>(w4.DRAW_COLORS, 1 * sizeof<u16>());
     w4.text("Hello from\nAssemblyScript!", 10, 10);
 
     const gamepad = load<u8>(w4.GAMEPAD1);
     if (gamepad & w4.BUTTON_1) {
-        store<u16>(w4.DRAW_COLORS, 4);
+        store<u16>(w4.DRAW_COLORS, 2 * sizeof<u16>());
```

Comment:
This shouldn't change I think. The value of "4" simply means the 4th palette color, and is independent of the size of u16.

Reply:
Are you sure? palette have 4 colors. Index starts from zero, so 4th element should have 3th index, not 4th